docs/blocks-dynamic.md

```diff
@@ -0,0 +1,77 @@
+It is also possible to create dynamic blocks. These are blocks that can change their content even if the post is not saved. One example from WordPress itself is the latest posts block. This block will update everywhere it is used whena new post is published.
+
+The following code example shows how to create the latest post block dynamic block.
+
+```js
```

Comment:
Per consistency with other docs, should we include ES5 and ESNext varieties here?